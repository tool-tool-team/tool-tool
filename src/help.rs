use crate::{NAME, VERSION, DESCRIPTION};

pub fn print_help() {
    println!("{} {}", NAME, VERSION);
    println!();
    println!("{}", DESCRIPTION);
    println!();
}
