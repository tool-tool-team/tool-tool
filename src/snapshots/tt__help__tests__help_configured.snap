---
source: src/help.rs
expression: help_text
---
"ðŸ”§ tool-tool 0.1.0 ðŸ”§\n\nðŸ”§ A light-weight meta-tool to version and install tool dependencies for your software projects\n\nðŸ”§ Loaded configuration from foo.bar.yaml\n\nUsage: tt [-v] <command> <args...>\n  Run tool <command> with the provided arguments\n\nFlags:\n  -v     Verbose debug output\n\nAvailable commands:\n\n   Command  Tool\n   apply    fizz 4.5.6\n   bar      foo 1.2.3\n   buzz     fizz 4.5.6\n"
